win32ss/reactx/ntddraw/d3d.c

```diff
@@ -289,15 +289,16 @@ DWORD
 APIENTRY
 NtGdiDdDestroyD3DBuffer(HANDLE hSurface)
 {
-    PGD_DXDDDESTROYD3DBUFFER pfnDdDestroyD3DBuffer = (PGD_DXDDDESTROYD3DBUFFER)gpDxFuncs[DXG_INDEX_DxDdDestroyD3DBuffer].pfn;  
-    
+    PGD_DXDDDESTROYD3DBUFFER pfnDdDestroyD3DBuffer =
+      (PGD_DXDDDESTROYD3DBUFFER)gpDxFuncs[DXG_INDEX_DxDdDestroyD3DBuffer].pfn;
```

Comment:
The file has 4 space indentation. If you want to reformat that line, like you did, you should use 2x4, not 2x3.
```suggestion
        (PGD_DXDDDESTROYD3DBUFFER)gpDxFuncs[DXG_INDEX_DxDdDestroyD3DBuffer].pfn;
```

Reply:
Done, though, personally, I find it a bit confusing to use same indentation for a continued line as for a separate one.